cli/azd/extensions/microsoft.azd.ai.builder/internal/cmd/start.go

```diff
@@ -342,11 +346,16 @@ func (a *startAction) Run(ctx context.Context, args []string) error {
 			return fmt.Errorf("failed to marshal app config: %w", err)
 		}
 
+		resourceId := ""
+		if 0 <= len(a.scenarioData.AppResourceIds) && len(a.scenarioData.AppResourceIds) >= (i+1) {
+			resourceId = a.scenarioData.AppResourceIds[i]
+		}
```

Comment:
I think we may need something like this, otherwise it looks like it's setting `existing: true` even if user selects Create new Container App/App Service

```suggestion
		if a.scenarioData.AppResourceIds[i] != "new" {
			resourceId = a.scenarioData.AppResourceIds[i]
		}
```